### DISCUSSION TITLE
Initial setup: Balance €0, did set an "initial balance"

### DISCUSSION THREAD

**Discussion by bernhardflokstra** [firefly-iii/firefly-iii#4923] [Firefly III]
I just set up firefly for the first time. I set up a main asset account and gave it an initial balance of €2400. My net worth says €2400, Accounts > Asset accounts says this specific account has a balance of €2400. On the dashboard the graph say €2400 too. Yet, above the graph it says "Balance €0,00." 

How do I get Firefly III to reflect the corrent overall balance?

**Comment by JC5** [comment]
The balance you see is the monthly in/out balance and does not include the initial balance.
